examples/seq2seq/finetune_trainer.py

```diff
@@ -252,45 +293,53 @@ def main():
         data_args=data_args,
     )
 
+    all_metrics = {}
     # Training
     if training_args.do_train:
+
+        t0 = time.time()
```

Comment:
```suggestion
        start_time = time.time()
```
(and below at L304 if accepted)